ros2bag/test/resources/empty_bag/metadata.yaml

```diff
@@ -13,13 +13,13 @@ rosbag2_bagfile_information:
         name: /parameter_events
         type: rcl_interfaces/msg/ParameterEvent
         serialization_format: cdr
-        offered_qos_profiles: "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
```

Comment:
I think this can be better written as using [YAML multiline](https://yaml-multiline.info/) strings

Reply:
I agree that this could be better formatted - but this is representative output of what rosbag2 actually writes today - I've filed https://github.com/ros2/rosbag2/issues/609 to followup on a better formatting